README.md

```diff
@@ -174,6 +174,12 @@ Available options:
                            Defaults to 8. Pass 0 to disable the limit. Also
                            configurable through
                            VAULTENV_MAX_CONCURRENT_REQUESTS.
+  --allow-duplicate-variables
```

Comment:
Since there are three sensible things that could be done, erroring, keeping existing variables unchanged, and overwriting existing variables, maybe having a three-state flag here would be clearer. Then you could have `--duplicate-variables <error|keep|overwrite>` - which are all more self-explanatory than "allow".